### DISCUSSION TITLE
How to check condition clickable

### DISCUSSION THREAD

**Discussion by jbarriojim** [mobile-dev-inc/Maestro#1583] [Q&A] [answered]
Hi,

In one of my workflows I'm repeating 10 times the same action because I cannot check that one element is clickable or not:

![imagen](https://github.com/mobile-dev-inc/maestro/assets/24703132/be428934-4bfc-4a07-8d67-23128a75d465)

When the element is in the minimum value, I don't need to decreasing again. The current conditions in maestro are [this ones](https://maestro.mobile.dev/advanced/conditions#conditions).

Is it any way to take advantage of the `true:` one to check if one element is clickable or not with Javascript? Could be this condition added to the list to be used?

Thank you in advance.

**Comment by simon-gilmurray** [comment]
Can you inspect the minus button in terminal using `Maestro Hierarchy`? It should have the `enabled: false` flag if it's not clickable, and the `enabled: true` flag if it is which you should be able to use to base your conditional off.

**Reply by jbarriojim** [reply]
Yes, it has that flag
![imagen](https://github.com/mobile-dev-inc/maestro/assets/24703132/a0d63624-d753-4f62-b91d-99a2285a7166)
![imagen](https://github.com/mobile-dev-inc/maestro/assets/24703132/7d393eb4-5b2c-476c-8157-658008b9c838)

How can I access to that flag to check it?

**Reply by simon-gilmurray** [reply, marked as answer]
You should be able to just use it conditionally, e.g.

```
- runFlow:
    when:
      visible: 
          id: <testID>
          enabled: true
     file: {reference to a javascript file}
```

https://maestro.mobile.dev/advanced/conditions

**Reply by jbarriojim** [reply]
Thank you! Many options are opened to me now 😄  😄
